web/custom-plugins/forecast-of-response/src/components/Result.tsx

```diff
@@ -218,6 +242,19 @@ export default (props: { result: any }) => {
       </div>
       <LinesOverTime data={chartData} graphInfo={graphInfo} />
       <ArrowPlots data={chartData} graphInfo={graphInfo} />
+      {index ? (
+        <></>
+      ) : (
+        <Button
+          style={{ width: '240px', marginLeft: '10px' }}
+          variant="outlined"
+          onClick={() => {
+            addResultGraph()
+          }}
+        >
+          Add extra result graph
```

Comment:
suggestion: change button name to "add extra graph window"

Reply:
Nice suggestion! Changed it accordingly.